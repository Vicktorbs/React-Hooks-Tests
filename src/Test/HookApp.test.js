import { shallow } from "enzyme"
import { HookApp } from "../HookApp"

describe('test on file HookApp', () => {
    
    test('should show <HookApp /> correctly', () => {
        const wrapper = shallow(<HookApp />);
        expect(wrapper).toMatchSnapshot();
    })
    
})
