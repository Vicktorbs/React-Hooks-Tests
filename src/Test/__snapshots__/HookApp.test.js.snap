// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test on file HookApp should show <HookApp /> correctly 1`] = `
<div>
  <h1>
    Hello world
  </h1>
</div>
`;
